lib/rubocop/directive_comment.rb

```diff
@@ -53,5 +48,31 @@ def range
     def match_captures
       @match_captures ||= comment.text.match(DIRECTIVE_COMMENT_REGEXP)&.captures
     end
+
+    def disabled?
```

Comment:
I'd add YARD metadata to all public methods, as this in a relative important public API class.

Reply:
Added